src/MLJMultivariateStatsInterface.jl

```diff
@@ -150,4 +147,1102 @@ metadata_pkg.(
+scatter matrix(`Pᵀ*Sb*P`) is maximized relative to the trace of the transformed
+within-class scatter matrix (`Pᵀ*Sw*P`).The projection matrix is scaled such that
+`Pᵀ*Sw*P=I` or `Pᵀ*Σw*P=I`(where `Σw` is the within-class covariance matrix) .
+Predicted class posterior probability for feature matrix `Xtest` are derived by
+applying a softmax transformationto a matrix `Pr`, such that  rowᵢ of `Pr` contains
+computed distances(based on a distance metric) in the transformed space of rowᵢ in
+`Xtest` to the centroid of each class.
+
+# Training data
+
+In MLJ or MLJBase, bind an instance `model` to data with
+
+    mach = machine(model, X)
+
+Where
+
+- `X`: is any table of input features (eg, a `DataFrame`) whose columns
+  are of scitype `Continuous`; check the scitypes with `schema(X)`
+- `y`: is the target, which can be any `AbstractVector` whose element
+  scitype is `OrderedFactor` or `Multiclass`; check the scitype
+  with `scitype(y)`
+
+Train the machine using `fit!(mach, rows=...)`.
+
+# Hyper-parameters
+
+- `method::Symbol=:gevd`: The solver, one of `:gevd` or `:whiten` methods.
+- `cov_w::CovarianceEstimator`=SimpleCovariance: An estimator for the within-class
+    covariance (used in computing within-class scatter matrix, Sw), by default set
+    to the standard `MultivariateStats.SimpleCovariance()` but
+    could be set to any robust estimator from `CovarianceEstimation.jl`.
+- `cov_b::CovarianceEstimator`=SimpleCovariance: The same as `cov_w` but for the between-class
+    covariance (used in computing between-class scatter matrix, Sb).
+- `out_dim::Int=0`: The output dimension, i.e dimension of the transformed space,
+    automatically set if 0 is given (default).
+- `regcoef::Float64=1e-6`: The regularization coefficient (default value 1e-6). A positive
+    value `regcoef * eigmax(Sw)` where `Sw` is the within-class scatter matrix, is added
+    to the diagonal of Sw to improve numerical stability. This can be useful if using
+    the standard covariance estimator.
+- `dist=Distances.SqEuclidean()`: The distance metric to use when performing
+   classification (to compare the distance between a new point and centroids in
+   the transformed space); must be a subtype of `Distances.SemiMetric` from
+   Distances.jl, e.g., `Distances.CosineDist`.
+
+# Operations
+
+- `transform(mach, Xnew)`: Return a lower dimensional projection of the input `Xnew` having the same scitype as `X` above.
+- `predict(mach, Xnew)`: Return predictions of the target given
+  features `Xnew` having the same scitype as `X` above. Predictions
+  are probabilistic but uncalibrated.
+- `predict_mode(mach, Xnew)`: Return the modes of the probabilistic predictions
+   returned above.
+
+
+# Fitted parameters
+
+The fields of `fitted_params(mach)` are:
+
+- `projected_class_means`: The matrix comprised of class-specific means as columns,
+   of size `(in_dim, nc)`, where `in_dim` is the number of input features (columns) and
```

Comment:
To support  #46, #47,  can you please change `in_dim` -> `indim` everywhere in documentation?

```suggestion
   of size `(indim, nc)`, where `in_dim` is the number of input features (columns) and
```